src/app/components/error-banner/error-banner.tsx

```diff
@@ -0,0 +1,11 @@
+import { ErrorSummary } from 'nhsuk-react-components';
```

Comment:
this component has a very general name (error-banner) but the content of the component is very situation-specific (validation errors on a page)